main.go

```diff
@@ -218,7 +218,8 @@ func buildBin(path string) error {
 	if err := runner.Run("go", "mod", "tidy"); err != nil {
 		return fmt.Errorf("'go mod tidy' failed: %w", err)
 	}
-	if err := runner.Run("go", "build", "-o", path, "."); err != nil {
+	if err := runner.Run("go", "build", "-buildvcs=false",
+	 "-o", path, "."); err != nil {
```

Comment:
I think this fails gofmt. Good reminder that I should put a gofmt check in my primary `op`.

In cases like these, I prefer to split the error assignment into its own line. I typically only do if-with-a-short-statement IFF it fits within 80col. It's kind of a sanity check on myself in that way.

```suggestion
	err := runner.Run("go", "build", "-buildvcs=false", "-o", path, ".")
	if err != nil {
```

Reply:
> I think this fails gofmt. Good reminder that I should put a gofmt check in my primary `op`.
> 
> In cases like these, I prefer to split the error assignment into its own line. I typically only do if-with-a-short-statement IFF it fits within 80col. It's kind of a sanity check on myself in that way.

Thank you for the suggestion, I pushed up the change breaking it into two lines.